src/ngraph/frontend/onnx_import/attribute.hpp

```diff
@@ -20,6 +20,9 @@
 #include "onnx.pb.h"
 #include "tensor.hpp"
 
+#define likely(__x)         __builtin_expect(!!(__x), 1)
```

Comment:
If we use `#define` in this header file, we will potentially be affecting code using this header.

Reply:
Only ONNX importer use this header, no one else. It not supposed to be used by others.